locales/en.default.json

```diff
@@ -108,6 +108,10 @@
       "choose_options": "Choose options",
       "choose_product_options": "Choose options for {{ product_name }}",
       "description": "Description",
+      "inventory_in_stock": "In stock",
+      "inventory_in_stock_show_count": "{{ quantity }} in stock",
+      "inventory_low_stock": "Low stock",
+      "inventory_low_stock_show_count": "Low stock: {{ quantity }} left",
```

Comment:
I wonder if it could be useful for a merchant to only display this block when the amount in stock is below the threshold. So only showing `Low stock: X left` as a way to push customers to action their purchase without showing the `In stock` version. 

It might be not great in some scenarios with some variants in stock and others with low stock but it could be something none the less. Some stores with single variant products, etc, could have a use for it. 

Just wanted to bring it up, not to change anything in this PR but to check if we thought about this specific use case. 

cc: @YoannJailin

Reply:
Hey @ludoboludo, that's a good point. I considered it as well in the beginning, but after trying that kind of block on our partner themes, most of them use that "In Stock" state by default. Made more sense while configuring the settings and especially the threshold.